src/miniflask/miniflask.py

```diff
@@ -101,10 +101,18 @@ def print_heading(self, *args, color=fg('blue'), margin=8):
         print(color + attr('bold') + line + attr('reset'))
 
     def print_recently_loaded(self, prepend="", loading_text=highlight_loading):
+        last = ""
+        last_formatted = ""
         for i, mod in enumerate(self._recently_loaded):
-            module_id = mod.miniflask_obj.module_id_initial
+            module_id = module_id_formatted = module_id = mod.miniflask_obj.module_id_initial
```

Comment:
Duplicated variable assignment `module_id = module_id = ...`.

Reply:
- `mf.register_parent(False)` would be possible but would require a complete rewrite of the registration process. The problem is, that the parents are loaded *before* the module registration gets started.
- I will recheck this and remove it before merging the `v2-dev` branch into master for all upcoming tests until then

Reply:
Sure, it's just a way the "user" might see it.

Reply:
I suspect that the function of automatic loading of the parents is used in 99.9% of cases. Until we have a better picture of the cases that require to explicitly disable this feature, I'm merging this PR into the v2-dev branch.